dockernet/config/relayer_config_dydx_noble.yaml

```diff
@@ -73,6 +73,14 @@ paths:
     src-channel-filter:
       rule: ""
       channel-list: []
+  stride-noble:
```

Comment:
In theory do we really need this relayer?  Is it just for the ICA initial setup?
We never send any direct messages to the reward chain, it is only done via pfm...

Reply:
right - only used for the ICA creation